components/web/squid/Makefile

```diff
@@ -133,9 +133,10 @@ COMPONENT_TEST_TRANSFORMS += \
 	'-e "/TOTAL/p" ' \
 	'-e "/SKIP/p" ' \
 	'-e "/PASS/p" ' \
+	'-e "/XPASS/p" ' \
 	'-e "/FAIL/p" ' \
-	'-e "/Can not open file/d" ' \
-	'-e "/ERROR/p" '
+	'-e "/XFAIL/p" ' \
+	'-e "/^\# ERROR/p" '
```

Comment:
/FAIL/p matches both FAIL and XFAIL lines. This is the reason you'll have two XFAIL entries in results-all.master now.